services-worker/src/main/java/edu/unc/lib/dl/cdr/services/imaging/ThumbnailEnhancement.java

```diff
@@ -146,8 +147,12 @@ public Element call() throws EnhancementException {
 
 	private void createStoreThumb(String dsIrodsPath, int width, int height, String dsname, boolean exists,
 			List<String> thumbRels) throws Exception {
+		long start = System.currentTimeMillis();
```

Comment:
would you mind naming these long vars more specifically within this method? e.g., imageStart, datastreamStart, cleanupStart